components/kubeflow/kfserving/README.md

```diff
@@ -0,0 +1,174 @@
+# KFServing Component
+
+This is the Kubeflow Pipelines component for KFServing. This uses the [V1beta1 API](https://github.com/kubeflow/kfserving/blob/master/docs/apis/v1beta1/README.md).
+
+## Usage
+
+Load the component with:
+
+```python
+import kfp.dsl as dsl
+import kfp
+from kfp import components
+
+kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kubeflow/kfserving/component.yaml')
```

Comment:
Thanks @pvaneck, do you mind adding a note that if someone wants to use the v1alpha2 api version, you can load the url using a commit link like
```
https://raw.githubusercontent.com/kubeflow/pipelines/65bed9b6d1d676ef2d541a970d3edc0aee12400d/components/kubeflow/kfserving/component.yaml
```

Reply:
Good call. I made a note of it under the code block.